service/src/app.py

```diff
@@ -41,8 +77,9 @@ def predict():
     Returns:
         JSON: A JSON object containing the prediction result.
     """
+    PREDICTION_COUNT.inc()
```

Comment:
Just want to double-check is the response is a POST request?

Reply:
I think you might be right and it might be a GET request. Initially we were passing the URL as a body which is usual for POST requests, but as it is now with params and keeping in mind the fact that it doesn't affect the state of the server, it should be a GET. Regardless, both options would work since the endpoint called in model service accepts both methods. Definetely it is something we should update though and ensure that changing it doesn't actually break anything.